pages/account-detail.vue

```diff
@@ -181,11 +181,13 @@ export default {
 
     prettyPrintAmount: function (row, column) {
       const prec = Math.pow(10, row.precision);
+      BigNumber.set({ DECIMAL_PLACES: 2 });
```

Comment:
@xpgeng I think it would be better if we set the decimal places not on the constructur but on **each** individual `BigNumber` object. Maybe setting it on the constructor will affect other instances. Unwanted side effects.
Please change the constructor call to `instance.decimalPlaces(2).toFixed()`